conda_smithy/configure_feedstock.py

```diff
@@ -471,7 +471,7 @@ def render_appveyor(jinja_env, forge_config, forge_dir):
         if os.path.exists(cfbs_fpath):
             fast_finish_script += "{recipe_dir}\\ff_ci_pr_build".format(recipe_dir=forge_config["recipe_dir"])
         else:
-            get_fast_finish_script += "curl https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/master/recipe/ff_ci_pr_build.py > ff_ci_pr_build.py"
+            get_fast_finish_script += '''powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/master/recipe/ff_ci_pr_build.py', 'ff_ci_pr_build.py')"'''
```

Comment:
> Why not the much cleaner and guaranteed to work `appveyor DownloadFile`.

Short answer people have tried it and they run into intermittent issues. The fix for their case is to use PowerShell like we are doing here. I'm not seeing similar reports with PowerShell.

ref: http://help.appveyor.com/discussions/problems/2753-rawgithubusercontentcom-gives-server-unavailable-sporadically

> ...or better yet, the solution we had before and was working without depending on any external tool.

This line has only ever used `curl`. Not sure what you are think of, but I'm pretty sure it is something else. Though please feel free to clarify.